models/src/main/protobuf/CasperMessage.proto

```diff
@@ -74,6 +74,9 @@ message BlockRequest {
   bytes  hash       = 2;
 }
 
+message ForkTipRequest {
+}
```

Comment:
Maybe fill this with your own fork choice tip and so if the other validator's fork choice tip is the same, then don't return anything. Doesn't have to be done now.

Reply:
Let's leave it for the future - especially that you mentioned the new block store eliminating the need for that in the first place. I guess I'll ticket a JIRA for removal of 'ForkChoiceTipRequest' after the new block store lands?